website/docs/r/advanced_cluster.html.markdown

```diff
@@ -14,6 +14,7 @@ More information on considerations for using advanced clusters please see [Consi
 
 ~> **IMPORTANT:**
 <br> &#8226; The primary difference between [`mongodbatlas_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cluster) and [`mongodbatlas_advanced_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster) is that `mongodbatlas_advanced_cluster` supports multi-cloud clusters.  We recommend new users start with the `mongodbatlas_advanced_cluster` resource.  
+<br> &#8226; A minimum oplog retention of 3 hours is required when seeking to change a cluster's class to Local NVMe SSD. To learn more see [`oplogMinRetentionHours`](https://www.mongodb.com/docs/manual/core/replica-set-oplog/#std-label-replica-set-minimum-oplog-size) 
```

Comment:
This type of warning would make the most sense in the https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster#advanced-configuration-options section where we explain the parameter https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster#oplog_size_mb this is where users would get burned if they scale up and that scale up then scales up the oplog but so Terraform will now have a value that doesn't match and will complain.  

Furthermore oplogMinRetentionHours isn't actually an option yet.   Please read the ticket you cited in our 1:1 notes.  Once we know how that will role out and where we can update Terraform, both code and docs.   I'd create a ticket in the backlog pointing to that and once it is deployed we can act on it.